Sources/SwiftSyntax/SyntaxVisitor.swift.gyb

```diff
@@ -86,7 +86,11 @@ open class SyntaxVisitor {
       }
       visitPost(node)
 %   else:
+%     if node.must_uphold_invariant:
+      let node = ${node.name}(data)!
```

Comment:
Could you also add a comment like the one I suggested for `SyntaxRewriter` here?

Reply:
Done.